src/TestCodeFiles/ConstructTests/Test1/Test1Driver.java

```diff
@@ -0,0 +1,15 @@
+package TestCodeFiles.ConstructTests.Test1;
+
+import NAR.Editor;
+import NAR.Parser;
+
+public class Test1Driver {
+    public static void main(String args[])
```

Comment:
This is perfectly ok but you might want to have main run the test for every construct in the same file. If you do decide to do that then you can rename the class to TestDriver